### DISCUSSION TITLE
ARM64 version for docker

### DISCUSSION THREAD

**Discussion by nyok92** [GaetanF/cyborgbackup#84] [General]
Hi @GaetanF ,
Would it be possible to have ARM64 version of images ?
I would like to make an [addon](https://developers.home-assistant.io/docs/add-ons/) for [home assistant](https://www.home-assistant.io/)
Thanks

**Comment by GaetanF** [comment]
Hello,

Did you request image only for CyBorgBackup or for both UI and Server ?

It's not better to directly use the Server API  ? Or maybe you want to run the server on RaspberryPI host ?

Regards,
